source/graphics/MapGraphic.js

```diff
@@ -89,14 +87,44 @@ export default class MapGraphic extends Graphic {
     this._generalBounds = [[Infinity, Infinity], [-Infinity, -Infinity]]
   }
 
-  /** Apply projectiong _before_ cartogram computation */
-  updatePreProjection() {
-    const projection = geoAlbersUsa()
-      .scale(canvasDimensions.width)
-      .translate([
-        canvasDimensions.width * 0.5,
-        canvasDimensions.height * 0.5,
-      ])
+  /** Apply projection _before_ cartogram computation */
+  updatePreProjection(geography) {
```

Comment:
would it be reasonable to store these projections elsehwere rather than hardcoding strings to match here? Maybe in GeographyResource.js?